prediction-polls/backend/src/repositories/ProfileDB.js

```diff
@@ -282,5 +282,30 @@ async function followerProfiles(userId) {
         return { error: errorCodes.DATABASE_ERROR };
     }
 }
+async function getRankPerTag(topic){
+    try {
+        const query = 'SELECT user.id , user.username has_domain_point.amount from user, has_domain_point where user.id = has_domain_point.userID AND has_domain_point.topic = ? Order By has_domain_point.amount Desc LIMIT 100';
```

Comment:
I think the sql is wrong. could u revise it?

Reply:
Fixed it
SELECT users.id , users.username, has_domain_point.amount from users, has_domain_point where users.id = has_domain_point.userId AND has_domain_point.topic = ? Order By has_domain_point.amount Desc LIMIT 100